### PR TITLE
Assert no errors are in pipeline tests

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/elastic-package/issues/281

### CODE REVIEW

File: internal/testrunner/runners/pipeline/runner.go [hunk 1]
```diff
@@ -311,6 +311,11 @@ func verifyDynamicFields(result *testResult, config *testConfig) error {
 func verifyFieldsInTestResult(result *testResult, fieldsValidator *fields.Validator) error {
 	var multiErr multierror.Error
 	for _, event := range result.events {
+		err := checkErrorMessage(event)
+		if err != nil {
+			multiErr = append(multiErr, err)
```
Comment (ycombinator) [inline internal/testrunner/runners/pipeline/runner.go:316]: Should we `continue` here? IOW, should we check fields in the event if we find the `error.message` field?

Comment (mtojek) [reply internal/testrunner/runners/pipeline/runner.go:316]: Hmm... maybe you're right. Actually all other fields can be wrong, e.g. should be removed but weren't due to an error.

Comment (mtojek) [reply internal/testrunner/runners/pipeline/runner.go:316]: Fixed

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**